test/unit/Pool/addAsset.js

```diff
@@ -63,11 +83,21 @@ describe('addAsset', function () {
   });
 
   it('should correctly add the asset with its min, max, and slippage ratio', async function () {
-    const { pool } = this;
+    const { pool, dai, stETH, chainlinkDAI, chainlinkSteth } = this;
 
     const ERC20Mock = artifacts.require('ERC20Mock');
     const token = await ERC20Mock.new();
 
+    const chainlinkNewAsset = await ChainlinkAggregatorMock.new();
+    await chainlinkNewAsset.setLatestAnswer(new BN((1e18).toString()));
+    const priceFeedOracle = await PriceFeedOracle.new(
+      [dai.address, stETH.address, token.address],
+      [chainlinkDAI.address, chainlinkSteth.address, chainlinkNewAsset.address],
+      [18, 18, 18],
+    );
+
+    await pool.updateAddressParameters(hex('PRC_FEED'), priceFeedOracle.address, { from: governance });
```

Comment:
The new price feed does not contain the asset that you added to the pool at the very top of this file (inside beforeEach). The solution would be that whenever we set a new price feed - the pool should check that it has aggregators for all its assets. This would also prevent us from pushing a bad address for the price feed (typo in address resulting in adding an EOA for the price feed that will always revert).